test/e2e/flowaggregator_test.go

```diff
@@ -776,7 +847,7 @@ func checkRecordsForFlowsClickHouse(t *testing.T, data *TestData, srcIP, dstIP,
 	assert.GreaterOrEqualf(t, len(clickHouseRecords), expectedNumDataRecords, "ClickHouse should receive expected number of flow records. Considered records: %s", clickHouseRecords)
 }
 
-func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcNodeName string, srcPodName string, srcIP string, dstIP string, dstPort int32, isIPv6 bool) {
+func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcNodeName string, srcPodName string, srcIP string, dstIP string, dstPort int32, isIPv6 bool, checkEgress bool, egressName, egressIP string) {
```

Comment:
the parameter list is starting to get a bit long. Maybe these `checkX` functions could use some refactoring in the future
In the mean time, maybe remove `checkEgress` and use `egressName != ""` in the function?